vyos/vyos-1x
# monitoring: T3872: Add new feature service monitoring telegraf

## Change Summary

Add new feature, `service monitoring telegraf`, VyOS 1.3

## Related PR ##
https://github.com/vyos/vyos-build/pull/216

## Types of changes


## Related Task(s)

* https://phabricator.vyos.net/T3872

## Component(s) name

monitoring, telegraf
## Proposed changes


## How to test
```
set service monitoring telegraf authentication organization 'vyos'
set service monitoring telegraf authentication token 'XXXX...=='
set service monitoring telegraf bucket 'bucket'
set service monitoring telegraf port '8086'
set service monitoring telegraf source 'all'
set service monitoring telegraf url 'http://foo.local'
```
Expected running `vyos-telegraf.service`
```
vyos@r4:~$ sudo systemctl status vyos-telegraf
● vyos-telegraf.service - The plugin-driven server agent for reporting metrics into InfluxDB
   Loaded: loaded (/etc/systemd/system/vyos-telegraf.service; disabled; vendor preset: enabled)
  Drop-In: /etc/systemd/system/vyos-telegraf.service.d
           └─10-override.conf
   Active: active (running) since Tue 2022-02-08 17:43:37 EET; 4min 39s ago
     Docs: https://github.com/influxdata/telegraf
 Main PID: 8074 (telegraf)
    Tasks: 12 (limit: 1150)
   Memory: 33.6M
   CGroup: /system.slice/vyos-telegraf.service
           └─8074 /usr/bin/telegraf -config /run/telegraf/vyos-telegraf.conf -config-directory /etc/telegraf/telegraf.d
```

## Checklist:
